blt/urls.py

```diff
@@ -412,15 +412,10 @@
     re_path(r"^ratings/", include("star_ratings.urls", namespace="ratings")),
     path("robots.txt", website.views.robots_txt),
     path("ads.txt", website.views.ads_txt),
-    re_path(r"^contributors/$", contributors_view, name="contributors"),
-    path("company/", include("company.urls")),
-    path("sponsor/", website.views.sponsor_view, name="sponsor"),
-    path("companies/", DomainListView.as_view(), name="domain_list"),
```

Comment:
What happened here?

Reply:
There were Conflicts here
should I add them back ?

Reply:
Yes please

Reply:
Done